src/__tests__/applicants.int.test.ts

```diff
@@ -134,16 +134,16 @@ describe('/applicants', () => {
           externalId: '🆔',
         })
         .expect(201);
-      const after = await getTableMetrics('applicants');
+      const after = await loadTableMetrics('applicants');
       logger.debug('after: %o', after);
-      expect(before.count).toEqual('0');
+      expect(before.count).toEqual(0);
```

Comment:
Just wanted to call this out since it is a minor but pleasing improvement: the ajv validation will validate AND coerce the results for us (so a valid number from the database will become a number rather than a string.)